src/hotspot/share/opto/superword.cpp

```diff
@@ -524,19 +524,6 @@ void SuperWord::mark_reductions() {
 //    extraction of scalar values from vectors.
 //
 bool SuperWord::SLP_extract() {
-
-#ifndef PRODUCT
-  if (_do_vector_loop && TraceSuperWord) {
-    tty->print("SuperWord::SLP_extract\n");
-    tty->print("input loop\n");
-    _lpt->dump_head();
-    _lpt->dump();
-    for (uint i = 0; i < _lpt->_body.size(); i++) {
-      _lpt->_body.at(i)->dump();
-    }
-  }
-#endif
-
```

Comment:
This was only printed if `_do_vector_loop` was on, i.e. if `OptionVectorize` enabled (kinda odd anyway).
And we already do `print_bb`, which prints all relevant nodes (enabled with `SW_INFO` or `TraceSuperWord`).